tests/playwright/src/runner/podman-desktop-runner.ts

```diff
@@ -336,11 +347,13 @@ export class PodmanDesktopRunner {
       mkdirSync(parentDir, { recursive: true });
     }
 
-    const settingsContent = JSON.stringify({
-      'preferences.OpenDevTools': 'none',
-      'extensions.autoCheckUpdates': this._autoCheckUpdate,
-      'extensions.autoUpdate': this._autoUpdate,
-    });
+    const settingsContent = this._customSettingsObject
+      ? JSON.stringify(this._customSettingsObject)
```

Comment:
Include the values we use in settings as default into default object for customSettingsObject. So we will get right of them in constructor.

Reply:
fixed